bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BObjectType.java

```diff
@@ -140,7 +145,7 @@ public void setMethods(MethodType[] methodTypes) {
         this.methodTypes = methodTypes;
     }
 
-    public void setInitializer(BMethodType initializer) {
+    public void setInitializer(MethodType initializer) {
```

Comment:
Shall we rename this to setInitMethod

Reply:
Since it's an internal method shouldn't it be okay to keep like that?